blog/data/authors/piotr.konkol.json

```diff
@@ -0,0 +1,5 @@
+{
+    "name": "Piotr Konkol",
+    "bio": "Junior Test Automation Developer",
```

Comment:
@pkonkol this bio is a little short (it is temporary right?). Also, `index.md` file is missing for a full profile support. Check the `README.md` for requirements.